From f7a7acc77707e481bc911c1607e92172bfa3e1bf Mon Sep 17 00:00:00 2001
From: Li Jun <b47624@freescale.com>
Date: Mon, 23 Jun 2014 15:39:35 +0800
Subject: [PATCH 0168/1691] MLK-11340-28 usb: gadget: add host_request_flag in
 usb_gadget for OTG

commit f7a7acc77707e481bc911c1607e92172bfa3e1bf from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

This patch adds host_request_flag in usb_gadget to store host request
information from application.

Signed-off-by: Li Jun <b47624@freescale.com>
(cherry picked from commit eeeb09c86010e72153fbe15b0b67eefd160a0344)
---
 include/linux/usb/gadget.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/usb/gadget.h b/include/linux/usb/gadget.h
index 5f65bd2..658fa4e 100644
--- a/include/linux/usb/gadget.h
+++ b/include/linux/usb/gadget.h
@@ -568,6 +568,7 @@ struct usb_gadget {
 	unsigned			b_hnp_enable:1;
 	unsigned			a_hnp_support:1;
 	unsigned			a_alt_hnp_support:1;
+	unsigned			host_request_flag:1;
 	unsigned			quirk_ep_out_aligned_size:1;
 	unsigned			is_selfpowered:1;
 };
-- 
1.9.1

